web/conf/context.jsonld

```diff
@@ -282,6 +282,9 @@
       "@id" : "http://purl.org/ontology/bibo/shortTitle",
       "@container" : "@set"
     },
+    "Sonstiges" : {
+      "@id" : "http://purl.org/lobid/lv#Miscellaneous"
+    },
```

Comment:
Miscellaneous is mapped twice: https://github.com/hbz/lobid-resources/blob/d7c972f12a6b7f4272fb43f8315183e5f360e8a3/web/conf/context.jsonld#L374-L376

Reply:
sonstige is correct

Reply:
Good catch ! Fixed with [99b1e54](https://github.com/hbz/lobid-resources/pull/1642/commits/99b1e5488936fadb7df371d5886ed65cb1a92496).